docs/books/ownCloud_iOS_App_Manual.adoc

```diff
@@ -8,5 +8,3 @@ The ownCloud Team <docs@owncloud.com>
 :toclevels: 1
 :icons: font
 :icon-set: octicon
-
-include::modules/ROOT/pages/index.adoc[leveloffset=+1]
```

Comment:
What gets removed here?

Reply:
The content there is generated in the shell script.